client/app/scripts/charts/nodes-chart.js

```diff
@@ -84,12 +84,29 @@ class NodesChart extends React.Component {
     const { panTranslateX, panTranslateY, zoomScale } = this.state;
     const transform = `translate(${panTranslateX}, ${panTranslateY}) scale(${zoomScale})`;
     const svgClassNames = this.props.isEmpty ? 'hide' : '';
+    const markerOffset = this.props.selectedNodeId ? '35' : '40';
+    const markerSize = this.props.selectedNodeId ? '10' : '30';
 
     return (
       <div className="nodes-chart">
         <svg
           width="100%" height="100%" id="nodes-chart-canvas"
-          className={svgClassNames} onClick={this.handleMouseClick}>
+          className={svgClassNames} onClick={this.handleMouseClick}
+        >
+          <defs>
+            <marker
+              className="edge-marker"
+              id="end-arrow"
+              viewBox="1 0 10 10"
+              refX={markerOffset}
+              refY="3.5"
+              markerWidth={markerSize}
+              markerHeight={markerSize}
+              orient="auto"
+            >
+              <polygon className="link" points="0 0, 10 3.5, 0 7" />
+            </marker>
```

Comment:
Maybe here you'd want to use `Marker`, as defined in `charts/marker.js`? Since only `markerOffset` and `markerSize` are variable, I'd just rename it to `EndArrowMarker` and put `children`, `id` and `viewBox` attributes inside there so that here you can simply have `<EndArrowMarker offset={markerOffset} size={markerSize} />`